pyfr/writers/vtk.py

```diff
@@ -138,6 +161,26 @@ def _get_npts_ncells_nnodes(self, sk):
 
         return npts, ncells, nnodes
 
+    def _get_npts_ncells_nnodes_ho(self, sk):
+        etype, neles = self.soln_inf[sk][0], self.soln_inf[sk][1][2]
+
+        if etype == 'pyr':
+            # No Lagrange pyr cells in VTK
+            # Therefore, rely on the subdivision mechanism
+            # of the vtk writer
+            return self._get_npts_ncells_nnodes(sk)
```

Comment:
I think we need to explicitly call the _div routine here.

Reply:
Good catch!